contracts/LendingPool.sol

```diff
@@ -158,15 +164,20 @@ contract LendingPool is OwnableUpgradeable, ERC721Upgradeable, Clone {
         if(sinceLoanStart > maxLoanLength){
             interest += ((sinceLoanStart - maxLoanLength)*borrowed)/(1 days);
         }
-        totalBorrowed -= borrowed;
+        unchecked {
+            totalBorrowed -= borrowed;
+        }
+        
         _burnWithoutBalanceChanges(loanId, from);
 
         if(sinceLoanStart < (1 days)){
             uint until24h;
+            uint toReduce;
             unchecked {
                 until24h = (1 days) - sinceLoanStart;
+                // Impossible to overflow since borrowed would overflow before this would
+                toReduce = (borrowed*until24h)/(1 days);
```

Comment:
llama sees reasoning, so you mean that until24h < sinceLoanStart * loan.interest?

Reply:
toReduce will have a maximum of 1.05x10^65 because: (2^216 * 86400) / 86400, which is under 1.15x10^77

Reply:
good point